src/main/java/duke/exception/DukeEmptyArgumentException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+import duke.exception.DukeException;
+
+public class DukeEmptyArgumentException extends DukeException {
+    @Override
+    public String toString() {
+        return "arguments shouldn't be empty";
+    }
+}
```

Comment:
I like how you made use of inheritance to represent different types of DukeExceptions rather than just using different Strings to represent different exceptions